packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/auth/EmailPasswordAuth.kt

```diff
@@ -87,18 +87,17 @@ public interface EmailPasswordAuth {
      */
     public suspend fun sendResetPasswordEmail(email: String)
 
-    // TODO https://github.com/realm/realm-kotlin/issues/744
-    // /**
-    //  * Call the reset password function configured to the
-    //  * [Credentials.Provider.EMAIL_PASSWORD] provider.
-    //  *
-    //  * @param email the email of the user.
-    //  * @param newPassword the new password of the user.
-    //  * @param args any additional arguments provided to the reset function. All arguments must
-    //  * be able to be converted to JSON compatible values using `toString()`.
-    //  * @throws AppException if the server failed to confirm the user.
-    //  */
-    // public suspend fun callResetPasswordFunction(email: String, newPassword: String, vararg args: Any?)
+    /**
+     * Call the reset password function configured to the
+     * [Credentials.Provider.EMAIL_PASSWORD] provider.
+     *
+     * @param email the email of the user.
+     * @param newPassword the new password of the user.
+     * @param args any additional arguments provided to the reset function. All arguments must
```

Comment:
I know this was in our old docs, but I don't think this is the case. The restriction is that the values can be converted to a BsonValue.

So maybe rewrite to something like 

```
@param args any additional arguments provided to the reset function. All arguments be able to be converted to a [BsonValue].
``